ts/project.ts

```diff
@@ -247,6 +268,20 @@ const proxyEnvironmentHelpers = (): ProjectHelpers => ({
     return templateLiteralFn;
   },
 
+  addPackage<T extends Project, Name extends string>(
+    this: T,
+    name: Name,
+    pkg: string,
+  ) {
+    const build = this.defaultEnvironment
+      ? this.build(pkg)
+      : emptyEnvironment.build(pkg);
+    return {
+      ...this,
+      ...({ [name]: build } as { [n in Name]: Package }),
+    };
+  },
+
```

Comment:
You didn't implement the backtick syntax that we still support for `Project.addCheck` & `Project.addExecutable`. I'm assuming that's not an oversight, but just something that we don't want to support for `Project.addPackage` right now, because in general we want a better API for that. Is that correct or is that just an oversight?

Reply:
Yeah that's kind of what I was thinking, but don't feel strongly. I was assuming the template literal syntax was being deprecated and therefore we shouldn't add to new methods, but let me know if you think we should be consistent until we fully remove it

Reply:
No, I think that's a good call!